ui/src/modules/plans/models/plan/plan-info.module.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 ABSA Group Limited
```

Comment:
@TerranceKhumalo-absa 2023

Reply:
Just a small picky note on copyright headers - very strictly speaking according to the [ABSA OSS guidelines](https://github.com/AbsaOSS/oss-guidelines#:~:text=The%20year%20in%20the%20copyright%20notice%20is%20the%20year%20of%20the%20inception%20of%20the%20projects) that should be a year of inception of the project. That is what was agreed on as a result of long debates. But practically, from the legal perspective (at our best knowledge) it doesn't matter too much which exact year is inserted as long as it's somewhere in between the project inception date and the current date. In either case the copyright notice has equal legal power (again, it's at our collective best knowledge, not officially confirmed).

What is more important however, is that it should **never be modified** once inserted.

In Spline, historically we simply insert the date of the file creation.

Reply:
No, of course you can change it in the PR. It should be considered sealed once the work is logically done on the file, e.g. after it's merged to the main branch. It's all complicated :)